admin_site/system/views.py

```diff
@@ -1874,3 +1895,50 @@ def get_context_data(self, **kwargs):
                 pc[pn] = pv
             pcs.append(pc)
         return pcs
+
+
+class ImageVersionsView(SiteMixin, SuperAdminOrThisSiteMixin, ListView):
+    """Image Versions are are sorted by major versions in the view
```

Comment:
This message has a typo (`are are`) and is not terribly explanatory. How about this?

```suggestion
    """Displays all of the image versions that this site has access to (i.e.,
    all versions released before the site's last_version datestamp).
```

Reply:
Thank you